src/test/java/org/folio/rest/ByIdTest.java

```diff
@@ -0,0 +1,88 @@
+package org.folio.rest;
+
+import static io.restassured.RestAssured.given;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.contains;
+
+import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.junit5.WireMockRuntimeInfo;
+import com.github.tomakehurst.wiremock.junit5.WireMockTest;
+import io.restassured.response.Response;
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+import org.folio.rest.tools.utils.NetworkUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(VertxExtension.class)
+@WireMockTest
+class ByIdTest {
```

Comment:
Is it better to create a completely new class just for this one test, when other tests for this endpoint are elsewhere (in `BLUsersAPITest`)? I appreciate the spirit of not adding to already bloated classes but breaking only the one out separately might make the code harder to follow.